index.js

```diff
@@ -27,8 +27,11 @@ module.exports = function conversation({name, app, appId,
   requestId = 'EdwRequestId.33ac9138-640f-4e6e-ab71-b9619b2c2210',
   locale = 'en-US',
   fixSpaces = false,
-  fuzzyDistance = 0.93
+  fuzzyDistance = 0.93,
+  handler = (app && app.handler) || null
 }) {
+  if(handler === null) throw new Error("Must provide either an app or handler.");
```

Comment:
@gamell Changed to single quotes to fix linter errors. As a side note, I noticed some other linter errors in the file - lots of missing trailing commas [comma-dangle], some missing parenthesis around arrow function arguments [arrow-parens], and an invalid `this`. If you like I can fix those as well (except for the invalid this, not sure how to fix that or even what that line is doing)